executor/oomtest/oom_test.go

```diff
@@ -223,8 +258,7 @@ func (h *oomCapture) Write(entry zapcore.Entry, fields []zapcore.Field) error {
 		return nil
 	}
 	// They are just common background task and not related to the oom.
-	if entry.Message == "SetTiFlashGroupConfig" ||
```

Comment:
Where is the original filter?

Reply:
The filter is a whitelist to filter available logs for the test. But I found that more and more logs need to be set in, and there will be conflicts between them. so I have to set the specifier filter at the test checker.